README.md

```diff
@@ -45,7 +45,7 @@ const transfer = TransferTransaction.create(
     'C5C55181284607954E56CD46DE85F4F3EF4CC713CC2B95000FA741998558D268',
     NetworkType.TEST_NET
   ),
-  [new Mosaic(new NamespaceId('cat.currency'), UInt64.fromUint(10000000))],
+  [new Mosaic(new NamespaceId('symbol.xym'), UInt64.fromUint(10000000))],
   PlainMessage.create('Welcome to NEM!'),
```

Comment:
Should change this message, too - 'Welcome to NEM!' to 'Welcome to Symbol.'

Reply:
Replaced all 'Welcome to NEM!' occurrences with 'Welcome to Symbol!' and updated all the copyrights, updated `README.md`.